libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/headline/PluggableHeadline.tsx

```diff
@@ -115,6 +122,13 @@ export class PluggableHeadline extends AbstractPluggableVisualization {
         return Promise.resolve(sanitizeFilters(newReferencePoint));
     }
 
+    public modifyInsightForDrillDown(
+        sourceVisualization: IInsight,
+        _drillDownContext: IDrillDownContext,
+    ): IInsight {
+        return sourceVisualization;
+    }
```

Comment:
hmm, perhaps this function should be moved & implemented in the AbstractPluggableVisualization? seems like a default no-op implementation. the superclass seems like an OK place for something like this.

Reply:
yep, added to AbstractPluggableVisualization